.gitattributes

```diff
@@ -56,3 +56,4 @@ tests/ export-ignore
 .scrutinizer.yml export-ignore
 .travis.yml export-ignore
 PULL_REQUEST_TEMPLATE export-ignore
+/libs/bower_components/angular-mocks export-ignore
```

Comment:
Is this lib required for development or why don't we remove it completely then?

Reply:
@sgiehl it seems to actually be required in some JS tests for dev. So it would just not be needed in the release.

Reply:
ah ok